docs/embedded/tutorials/migrate-abs-to-spe.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Sample App to Migrate from Azure Blob Storage container to SharePoint Embedded container
+description: Tutorial in how to migrate from Azure Blob Storage container to SharePoint Embedded container
+ms.date: 07/31/2024
+ms.localizationpriority: high
+---
+
+# Tutorial For Migrating Content From ABS To SPE
+
+## Purpose
+
+This tutorial will guide you through migrating content from Azure Blob Storage (ABS) to SharePoint Embedded (SPE). This is particularly useful for customers who have 500 docs in the blob storage container.
+
+### Prerequisites
```

Comment:
Maybe we can mention that they will need a M365 subscription, and an Azur e account. Or we are assuming they will figure it out from the SPE container and ABS container points below ?